ksqldb-functional-tests/src/test/resources/query-validation-tests/multi-col-keys.json

```diff
@@ -345,6 +345,26 @@
         "type": "io.confluent.ksql.util.KsqlStatementException",
         "message": "Invalid join condition: stream-table joins require to join on the table's primary key. Got S.K = T.K."
       }
+    },
+    {
+      "name": "multi-key insert - AVRO",
```

Comment:
what does this test that isn't already tested? I don't see any `INSERT` in this test?

Reply:
I was under the wrong impression input was doing insertion. Does QTT support insertion? Or I should use YATT?

Reply:
QTT does not test INSERT VALUES. RQTT does, though: https://github.com/confluentinc/ksql/blob/master/ksqldb-functional-tests/src/test/resources/rest-query-validation-tests/insert-values.json